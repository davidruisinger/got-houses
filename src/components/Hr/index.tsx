import styled from '../_utils/styledComponents'

interface Props {
  width?: string
}

const Hr = styled.hr<Props>`
  border: 0;
  background-color: ${props => props.theme.text_default};
  height: 1px;
  width: ${props => props.width};
`

Hr.defaultProps = {
  width: '100%',
}

export default Hr
