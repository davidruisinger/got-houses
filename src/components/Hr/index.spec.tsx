import 'jest-styled-components'
import React from 'react'
import testRenderer from 'react-test-renderer'
import Hr from './index'

const tree = testRenderer.create(<Hr />).toJSON()

describe('<Hr />', () => {
  it('matches the snapshot', () => {
    expect(tree).toMatchSnapshot()
  })
})
