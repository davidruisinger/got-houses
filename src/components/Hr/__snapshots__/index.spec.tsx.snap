// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hr /> matches the snapshot 1`] = `
.c0 {
  border: 0;
  height: 1px;
  width: 100%;
}

<hr
  className="c0"
  width="100%"
/>
`;
